tests/test-utils.js

```diff
@@ -3,11 +3,16 @@ import { createLocalVue, shallowMount, mount } from '@vue/test-utils'
 import Vuex from 'vuex'
 import { storeRaw } from '@/store'
 import Vuetify from 'vuetify'
-import wait from 'waait'
 
 const removeExtraSpaces = str => str.replace(/  +/g, ' ')
 const removeTabs = str => str.replace('\t', '')
 export const cleanString = str => removeTabs(removeExtraSpaces(str)).trim()
+export const wait = ms => {
+  if (process.env.CI !== "1" || process.env.INTEGRATION == "1") 
```

Comment:
At some point you should probably just remove the `waits` in the unit tests, so that we don't need this